Formula/nuclei.rb

```diff
@@ -0,0 +1,46 @@
+class Nuclei < Formula
+  desc "HTTP/DNS scanner configurable via YAML templates"
+  homepage "https://nuclei.projectdiscovery.io/"
+  url "https://github.com/projectdiscovery/nuclei/archive/v2.2.0.tar.gz"
+  sha256 "c503b565230f5375339c72872d496137abf8be26c156e20447d1ae0752169e40"
+  license "MIT"
+  head "https://github.com/projectdiscovery/nuclei.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GO111MODULE"] = "on"
+    cd "v2/cmd/nuclei" do
+      system "go", "build", "-o", "nuclei", "main.go"
```

Comment:
```suggestion
      system "go", "build", "main.go", *std_go_args
```

Reply:
build fails on my system with suggested change

Reply:
You also need to remove the `bin.install` line. See my suggested change below.

Reply:
missed this, thanks